docs/ref/models/relations.txt

```diff
@@ -56,6 +56,14 @@ Related objects reference
         some custom logic when a relationship is created, listen to the
         :data:`~django.db.models.signals.m2m_changed` signal.
 
+        .. versionchanged:: 1.9
+
+            The ``add()`` method executes one query regardless of the number
+            of objects being added, (unless the objects have not yet been saved
+            to the database in which case that is done). In prior versions, it
```

Comment:
I'd put the opening parenthesis before "in which case" instead of directly after the comma.